src/hud.cpp

```diff
@@ -158,24 +158,30 @@ void Hud::drawItem(const ItemStack &item, const core::rect<s32>& rect, bool sele
 	}
 
 //NOTE: selectitem = 0 -> no selected; selectitem 1-based
-void Hud::drawItems(v2s32 upperleftpos, s32 itemcount, s32 offset,
-		InventoryList *mainlist, u16 selectitem, u16 direction)
+void Hud::drawItems(v2s32 upperleftpos, s32 itemcount, s32 inv_offset,
+		InventoryList *mainlist, u16 selectitem, u16 direction, v2s32 offset)
```

Comment:
This should probably be `const v2s32 &offset`  if it's not modified by the function.